blockchain/fundamentals/ethereum-ii/erc721-tokens.md

```diff
@@ -0,0 +1,82 @@
+---
+author: mihaiberq
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+standards:
+  blockchain.compare-altcoins.1: 10
+  blockchain.compare-altcoins.5: 20
+
+aspects:
+
+  - introduction
+
+---
+# ERC721 Tokens
+
+---
+## Content
+
+**ERC721** tokens, on the other hand, are better suited for things you can not have 0.5 of, say cryptokitties[1]. ERC721 tokens are assumed to be unique and indivisible. You can only move them round in whole units, each one having a unique ID. They are known as *non-fungible tokens*.
+
+To better understand the difference, you can imagine that the main `address => uint256` mapping links an owner and the unique ID of something. The ID must be generated deterministically, usually by hashing object's properties.
```

Comment:
compared to what mapping in ERC20?

Reply:
uint is unique ID? isn't it unsigned integer of 256 bytes ?

Reply:
which object's properties?

Reply:
described erc20 mapping.

uint256 is an unsigned integer of 256 bytes indeed

reformulated the whole paragraph